docs/source/markdown/podman-manifest-add.1.md

```diff
@@ -79,6 +79,20 @@ Specify the variant which the list or index records for the image.  This option
 is typically used to distinguish between multiple entries which share the same
 architecture value, but which expect different versions of its instruction set.
 
+## Transport
+
+ Multiple transports are supported:
```

Comment:
What one is the default?

Reply:
I believe it's a [registry](https://github.com/containers/podman/blob/00792f8c95f804c446f8ae8e0e105a240074198e/docs/source/markdown/podman-push.1.md#destination)

Reply:
I just verified this. Building an image locally with localhost/myImage, then trying to add it with `manifest add mylist:v1 localhost/myImage` throws an error that `docker://localhost/myImage` yields in a connection refused on `https://localhost/v2/`

Reply:
Sorry, I meant the documentation should identify the the default transport.